src/System.Net.Security/src/System/Net/Security/NegotiateStream.cs

```diff
@@ -118,17 +118,45 @@ public void EndAuthenticateAsClient(IAsyncResult asyncResult)
 #endif
         }
 
-        public IAsyncResult BeginAuthenticateAsServer(AsyncCallback asyncCallback, object asyncState)
+        public virtual void AuthenticateAsServer()
+        {
+            AuthenticateAsServer((NetworkCredential)CredentialCache.DefaultCredentials, null, ProtectionLevel.EncryptAndSign, TokenImpersonationLevel.Identification);
+        }
+
+        public virtual void AuthenticateAsServer(ExtendedProtectionPolicy policy)
+        {
+            AuthenticateAsServer((NetworkCredential)CredentialCache.DefaultCredentials, policy, ProtectionLevel.EncryptAndSign, TokenImpersonationLevel.Identification);
+        }
+
+        public virtual void AuthenticateAsServer(NetworkCredential credential, ProtectionLevel requiredProtectionLevel, TokenImpersonationLevel requiredImpersonationLevel)
+        {
+            AuthenticateAsServer(credential, null, requiredProtectionLevel, requiredImpersonationLevel);
+        }
+
+        public virtual void AuthenticateAsServer(NetworkCredential credential, ExtendedProtectionPolicy policy, ProtectionLevel requiredProtectionLevel, TokenImpersonationLevel requiredImpersonationLevel)
```

Comment:
There are tests already, but they were using AuthenticateAsServerAsync in the sync case.  I fixed that, see NegotiateStreamStreamToStreamTest.cs.